src/tests/architect_tests/test_builders.py

```diff
@@ -108,135 +105,135 @@
+    [1, "2016-03-01", "3 month", "ems", "binary", 0],
+    [1, "2016-04-01", "3 month", "ems", "binary", 0],
+    [1, "2016-05-01", "3 month", "ems", "binary", 0],
+    [2, "2016-01-01", "3 month", "booking", "binary", 0],
+    [2, "2016-02-01", "3 month", "booking", "binary", 0],
+    [2, "2016-03-01", "3 month", "booking", "binary", 1],
+    [2, "2016-04-01", "3 month", "booking", "binary", 0],
+    [2, "2016-05-01", "3 month", "booking", "binary", 1],
+    [2, "2016-01-01", "3 month", "ems", "binary", 0],
+    [2, "2016-02-01", "3 month", "ems", "binary", 0],
+    [2, "2016-03-01", "3 month", "ems", "binary", 0],
+    [2, "2016-04-01", "3 month", "ems", "binary", 0],
+    [2, "2016-05-01", "3 month", "ems", "binary", 1],
+    [3, "2016-01-01", "3 month", "booking", "binary", 0],
+    [3, "2016-02-01", "3 month", "booking", "binary", 0],
+    [3, "2016-03-01", "3 month", "booking", "binary", 1],
+    [3, "2016-04-01", "3 month", "booking", "binary", 0],
+    [3, "2016-05-01", "3 month", "booking", "binary", 1],
+    [3, "2016-01-01", "3 month", "ems", "binary", 0],
+    [3, "2016-02-01", "3 month", "ems", "binary", 0],
+    [3, "2016-03-01", "3 month", "ems", "binary", 0],
+    [3, "2016-04-01", "3 month", "ems", "binary", 1],
+    [3, "2016-05-01", "3 month", "ems", "binary", 0],
+    [3, "2016-05-01", "3 month", "ems", "binary", 0],
+    [4, "2016-01-01", "3 month", "booking", "binary", 0],
+    [4, "2016-02-01", "3 month", "booking", "binary", 0],
+    [4, "2016-03-01", "3 month", "booking", "binary", 1],
+    [4, "2016-04-01", "3 month", "booking", "binary", 0],
+    [4, "2016-05-01", "3 month", "booking", "binary", 1],
+    [4, "2016-01-01", "3 month", "ems", "binary", 0],
+    [4, "2016-02-01", "3 month", "ems", "binary", 0],
+    [4, "2016-03-01", "3 month", "ems", "binary", 0],
+    [4, "2016-04-01", "3 month", "ems", "binary", 0],
+    [4, "2016-05-01", "3 month", "ems", "binary", 1],
+    [5, "2016-01-01", "3 month", "booking", "binary", 0],
+    [5, "2016-02-01", "3 month", "booking", "binary", 0],
+    [5, "2016-03-01", "3 month", "booking", "binary", 1],
+    [5, "2016-04-01", "3 month", "booking", "binary", 0],
+    [5, "2016-05-01", "3 month", "booking", "binary", 1],
+    [5, "2016-01-01", "3 month", "ems", "binary", 0],
+    [5, "2016-02-01", "3 month", "ems", "binary", 0],
+    [5, "2016-03-01", "3 month", "ems", "binary", 0],
+    [5, "2016-04-01", "3 month", "ems", "binary", 1],
+    [5, "2016-05-01", "3 month", "ems", "binary", 0],
 ]
 
-label_name = 'booking'
-label_type = 'binary'
+label_name = "booking"
+label_type = "binary"
 
 db_config = {
-    'features_schema_name': 'features',
-    'labels_schema_name': 'labels',
-    'labels_table_name': 'labels',
-    'sparse_state_table_name': 'staging.sparse_states',
+    "features_schema_name": "features",
+    "labels_schema_name": "labels",
+    "labels_table_name": "labels",
+    "sparse_state_table_name": "staging.sparse_states",
```

Comment:
I actually kind of prefer single-quoting for most things that aren't, say, English sentences.

This may in part be a hold-over from Ruby, where there's a syntactic difference. And, in Python, it's like a way of treating such things as "symbols" as opposed to "free text."

:man_shrugging:

Reply:
Postgres queries are nicer when we use double-quotes, I think, and a ton of our quoted strings in Triage are queries.

Reply:
100% agreed. It's a problem of which the automated tool defaults to, I understand.

As with a lot of these, I would hope we don't enforce these expectations on future code. But for this effort, it makes sense to me.